pydeconz/__init__.py

```diff
@@ -199,3 +201,21 @@ def async_event_handler(self, event: dict):
 
         else:
             _LOGGER.debug('Unsupported event %s', event)
+
+    def sync_group_color(self):
```

Comment:
Sync is a poor use of wording, I was confused right now since normally you specify async, maybe better to have it named update_group_color or similar